### ISSUE TITLE
Tests against real servers

### ISSUE THREAD

**Issue by zopyx** [PyFilesystem/pyfilesystem2#40]
For the new WebDAV driver we have tests that run against real webdav servers. We use Docker for maintaining various WebDAV servers. The nice thing is that you can use Docker images on Travis.
Wouldn't it make sense to use the same approach for at least the FTP tests?

**Comment by willmcgugan** [comment]
Yes, I think that would be a great idea. It's not something I could work on the the short term. Is that something you would want to contribute?

**Comment by zopyx** [comment]
The WebDav related tests are now running successfully on Travis using Docker

https://github.com/PyFilesystem/pyfilesystem2/blob/master/.travis.yml

https://github.com/PyFilesystem/webdavfs/blob/master/Makefile
